docs/onboarding_admins.rst

```diff
@@ -0,0 +1,82 @@
+Onboard Additional Admins
+=========================
+
+If you are the only admin for your SecureDrop, you can skip this chapter.
+It instructs you how to create additional *Admin Workstation* USB drives.
+Each *Admin Workstation* will have its own SSH keypair.
+
+This chapter assumes that you have one working *Admin Workstation*. If you've
+not completed that part of the setup yet, see :doc:`set_up_admin_tails`.  If
+your *Admin Workstation* is corrupted or lost, and you don't have a
+:doc:`backup <backup_workstations>`, see :doc:`rebuild_admin`.
+
+.. important::
+
+   If you make configuration changes on your servers using one
+   *Admin Workstation*, they may be overwritten by another *Admin Workstation*
+   if its local copy of the configuration is not identical. When working
+   with multiple admins, it is therefore important to establish protocols
+   for coordinating configuration changes. See :ref:`multiple_admins`.
+
+To onboard an additional administrator, you will need:
+
+- your existing *Admin Workstation* USB drive (referred to as **AW1** below)
+- an additional empty USB drive (referred to as **AW2** below)
+
+To set up AW2, follow these steps:
+
+1. Boot AW1, unlock its persistent volume, and set an admin password on the welcome screen.
+2. Ensure that Tails and the SecureDrop version on AW1 are up-to-date. If not, update now.
+3. Insert AW2.
+4. Launch the Tails installer (**Applications ▸ Tails ▸ Tails Installer**).
+5. Install Tails on AW2. This will delete all data on the AW2 USB drive.
+6. Shut down AW1.
+7. Boot AW2.
+8. Configure its persistent volume (**Applications ▸ Tails ▸ Configure persistent volume**).
+   Set a unique passphrase for AW2 and record it securely. Enable all persistence options.
+9. Reboot AW2, unlock its persistent volume, and set an admin password on the welcome screen.
+10. Insert AW1 and unlock it using the file manager (**Applications ▸ Accessories ▸ Files**).
+11. In a terminal, type: ``rsync -a /media/amnesia/TailsData/Persistent/securedrop ~/Persistent``
+
+    This will copy *only* the ``securedrop`` directory from AW1 to AW2.
+12. Generate a new keypair on AW2 using ``ssh-keygen -t rsa -b 4096``.
```

Comment:
The main reason for this bit of USB juggling is to minimize any risk of _accidental_ proliferation of this SSH keypair.